db/seeds.rb

```diff
@@ -18,3 +18,41 @@
 admin.save(validate: false)
 
 FactoryBot.create(:match)
+
+# Some example user entries
+
+FactoryBot.build(:user,
```

Comment:
Please do not use factorybot. It should be used for creating test factories... Just use `User.create`

Reply:
https://robots.thoughtbot.com/factory_girl-for-seed-data